app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -27,6 +33,17 @@ class EventDetailsViewModel(private val eventService: EventService, private val
     private val mutableEvent = MutableLiveData<Event>()
     val event: LiveData<Event> = mutableEvent
 
+    fun fetchEventSpeakers(id: Long) {
+        speakerService.fetchSpeakersForEvent(id)
+            .subscribeOn(Schedulers.io())
```

Comment:
@iamareebjamal subscribed 😅

Reply:
Nope

Reply:
Read about RxJava, this is not subscribing

Reply:
@iamareebjamal sorry for the delay i got confused a little bit